buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1389,5 +1405,42 @@ artifactId=${project.name}
         args argsNeeded
       }
     }
+
+
+    /** ***********************************************************************************************/
+
+    // Method to create the PortableValidatesRunnerTask.
+    project.ext.createPortableValidatesRunnerTask = {
```

Comment:
Can it be made more clear that the parameter for this closure is supposed to be a PortableValidatesRunnerConfiguration? While reading the code it took me a while to figure out that this function does expect that class as a parameter. If there's no downside to doing so, maybe use an explicit parameter like so:

```
project.ext.createPortableValidatesRunnerTask = { PortableValidatesRunnerConfiguration it -> ... }
```

Reply:
Comment sounds good to me.